Assets/SEE/GraphProviders/SingleGraphPipelineProvider.cs

```diff
@@ -100,19 +96,17 @@ protected override void RestoreAttributes(Dictionary<string, object> attributes)
                     foreach (object item in items)
                     {
                         Dictionary<string, object> dict = (Dictionary<string, object>)item;
-                        GraphProvider provider = RestoreProvider(dict);
+                        SingleGraphProvider provider = SingleGraphProvider.RestoreProvider(dict);
                         Pipeline.Add(provider);
                     }
                 }
                 catch (InvalidCastException e)
                 {
                     throw new InvalidCastException("Types are not assignment compatible."
-                        + $" Expected type: IList<{typeof(GraphProvider)}>. Actual type: {v.GetType()}."
-                        + $" Original exception: {e.Message} {e.StackTrace}");
+                                                   + $" Expected type: IList<{typeof(MultiGraphProvider)}>. Actual type: {v.GetType()}."
```

Comment:
Shouldn't this be `SingleGraphProvider`?

Reply:
I fixed it.